plugins/homepage/plugin_tests/homepageSpec.js

```diff
@@ -0,0 +1,110 @@
+/* globals girder, girderTest, describe, expect, it, runs, waitsFor */
+
+girderTest.addCoveredScripts([
+    '/clients/web/static/built/plugins/homepage/plugin.min.js'
+]);
+
+girderTest.startApp();
+
+function _goToHomepagePluginSettings() {
+    waitsFor(function () {
+        return $('a[g-target="admin"]:visible').length > 0;
+    }, 'admin console link to display');
+
+    runs(function () {
+        $('a[g-target="admin"]:visible').click();
+    });
+
+    waitsFor(function () {
+        return $('.g-plugins-config:visible').length > 0;
+    }, 'admin console to display');
+
+    runs(function () {
+        $('.g-plugins-config:visible').click();
+    });
+
+    waitsFor(function () {
+        return $('a[g-route="plugins/homepage/config"]:visible').length > 0;
+    }, 'plugins page to display');
+
+    runs(function () {
+        $('a[g-route="plugins/homepage/config"]:visible').click();
+    });
+
+    waitsFor(function () {
+        return $('.g-homepage-container:visible').length > 0;
+    }, 'homepage config to display');
+
+    waitsFor(function () {
+        return girder.rest.numberOutstandingRestRequests() === 0;
+    }, 'rest requests to finish');
+}
+
+function _verifyMarkdownContent(elem) {
+    expect(elem.find('p:contains("It\'s very easy")').length).toBe(1);
+    expect(elem.find('strong:contains("bold")').length).toBe(1);
+    expect(elem.find('em:contains("italic")').length).toBe(1);
+    expect(elem.find('a[href="https://girder.readthedocs.io/"]:contains("link to Girder!")').length).toBe(1);
```

Comment:
For what it's worth, there is a `toContain()` jasmine function that we could use instead of using the selector + length property.